corehq/apps/cloudcare/templates/form_entry/templates.html

```diff
@@ -310,9 +310,12 @@ <h3 class="caption" data-bind="html: header" tabindex="0"></h3>
       </div>
     </div>
     <div class="panel-footer">
-      <button class="btn btn-default btn-xs add" href="#"
-              data-bind="click: newRepeat, text: getTranslation('repeat.dialog.add.new', '{% trans_html_attr 'Add new repeat' %}')"
-              id="repeat-add-new" />
+      <button class="btn btn-default add" href="#"
+              data-bind="click: newRepeat"
+              id="repeat-add-new">
+              <i class="fa fa-plus"></i>
+              <!-- ko text: getTranslation('repeat.dialog.add.new', '{% trans_html_attr 'Add new repeat' %}') --><!-- /ko -->
```

Comment:
Could this comment be removed?

Reply:
That's knockout "containerless syntax": https://knockoutjs.com/documentation/text-binding.html

It performs slightly better than creating a span that just has a text binding.